client/view/workbook/settings/settings.js

```diff
@@ -0,0 +1,16 @@
+import settings from './settings.html'
+
+export default mediator => ({
+	name: `settings`,
```

Comment:
I would actually recommend making this new state a child of the `workbook` state, alongside the `workbook.select-sheet` and `workbook.sheet` states.

That way, it will automatically inherit the data from the `workbook` state's `resolve` function, along with some other benefits like if you switch from the `workbook.settings` state to the `workbook.sheet` state, it won't have to reload any of that workbook data.